src/main/java/com/icthh/xm/ms/balance/service/tenant/TenantService.java

```diff
@@ -38,6 +51,8 @@ public void createTenant(String tenant) {
 
         try {
             tenantListRepository.addTenant(tenantKey);
+            String content = IOUtils.toString(new ClassPathResource("/config/balancespec.yml").getInputStream(), UTF_8);
```

Comment:
Or maybe implement logic, when not found, then show just the empty data and don't throw Exception?

Reply:
not found not impossible
because this file inside source code =))